Sentry.xcodeproj/project.pbxproj

```diff
@@ -5219,10 +5230,11 @@
 				GCC_C_LANGUAGE_STANDARD = "compiler-default";
 				GCC_OPTIMIZATION_LEVEL = 0;
 				GCC_WARN_SHADOW = YES;
-				INFOPLIST_FILE = Sources/Sentry/Info.plist;
+				INFOPLIST_FILE = Sources/Resources/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				LD_DYLIB_INSTALL_NAME = "$(DYLIB_INSTALL_NAME_BASE:standardizepath)/$(EXECUTABLE_PATH)";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				MODULEMAP_FILE = "$(SRCROOT)/Sources/Resources/Sentry.modulemap";
```

Comment:
Just hit a build error locally because while this is the correct path, it shouldn't have been updated here. We should've updated the path at https://github.com/getsentry/sentry-cocoa/blob/3437454585c7d7f684776118db34b9018d83941c/Sources/Configuration/Sentry.xcconfig#L7

We really need to get all the build settings out of the pbxproj into xcconfigs and write a linter that makes sure we never add new settings to the pbxproj.

Reply:
fixed in https://github.com/getsentry/sentry-cocoa/pull/3315